tensorflow/python/compiler/tensorrt/test/testdata/gen_tf_variablev2_model.py

```diff
@@ -0,0 +1,63 @@
+import tensorflow as tf
+import tensorflow.compat.v1 as tf1
```

Comment:
These imports will most likely not work in the test environment. Below you will find suggestions how to replace them. Or is this file for reference only (i.e. it does not need to run as part of a test)? In the latter case the current format is probably fine.

```suggestion
from tensorflow.python.framework import ops
from tensorflow.python.ops import variable_scope
```

Reply:
This file is for reference only, the test only has the model files generated by this script as a dependency.